image/templates/helm/stackrox-central/templates/01-central-12-central-db.yaml

```diff
@@ -92,6 +92,10 @@ spec:
         volumeMounts:
         - name: disk
           mountPath: /var/lib/postgresql/data
+        {{- if ._rox.central.db.persistence.backup }}
```

Comment:
Please follow [the guide](https://github.com/stackrox/stackrox/blob/master/image/templates/CHANGING_CHARTS.md#add-new-values-field-to-stackrox-helm-chart) when adding a new user-tweakable helm value.
Alternatively (if you don't want to expose this to users) you should be able to use `[< if .Operator >] ... [<end>]` meta-templating.

Reply:
Didn't know about this `[<if .Operator >]` trick, thanks.

Reply:
I've wrapped it like this:

```yaml
        [< if .Operator >]
        {{- if ._rox.central.db.persistence.backup }}
        - name: backup
          mountPath: /var/lib/postgresql/backup
        {{- end }}
        [<end>]
```

But this doesn't seem to work:
```
"YAML parse error on stackrox-central-services/templates/01-central-12-central-db.yaml: error converting YAML to JSON: yaml: line 94: could not find expected ':'"
```

Anything I'm missing?

Reply:
Oh, sorry, this needs a `.htpl` extension to work. Perhaps instead of renaming check if `$._rox.env.installMethod` (which is set using this trick) equals `"operator"`.